lib/Conversion/ImportVerilog/Structure.cpp

```diff
@@ -90,6 +171,43 @@ struct MemberVisitor {
     return success();
   }
 
+  // Handle nets.
+  LogicalResult visit(const slang::ast::NetSymbol &netNode) {
+    auto loweredType = context.convertType(*netNode.getDeclaredType());
+    if (!loweredType)
+      return failure();
+
+    Value assignment;
+    if (netNode.getInitializer()) {
+      assignment = context.convertExpression(*netNode.getInitializer());
+      if (!assignment)
+        return failure();
+
+      if (assignment.getType() != loweredType) {
```

Comment:
Like bit a = 1'b0; will throw an error: `moore.bit` and `moore.packed<range<bit 0:0>>` type mismatch.